src/tools/compiletest/src/header.rs

```diff
@@ -298,6 +298,8 @@ pub struct TestProps {
     pub error_patterns: Vec<String>,
     // Extra flags to pass to the compiler
     pub compile_flags: Vec<String>,
+    // Extra flags to pass to rustdoc
+    pub rustdoc_flags: Vec<String>,
```

Comment:
This should be unnecessary, rustdoc tests already use `compile-flags` for this purpose.

Reply:
We need to pass `--document-private-items` to `rustdoc`, which is not supported by `rustc`.

Reply:
There is a bunch of existing rustdoc tests that use `compile-flags: --document-private-items` for doing exactly that.

Reply:
Oops, I missed that. I'll revert the changes to `compiletest`.